cmd/osde2ectl/create/cmd.go

```diff
@@ -150,20 +150,25 @@ func run(cmd *cobra.Command, argv []string) error {
 		}
 	}
 
-	if _, err := os.Stat(args.awsAccountsFile); err == nil {
-		csvfile, err := os.Open(args.awsAccountsFile)
-		if err != nil {
-			log.Fatalln("Couldn't open the csv file", err)
-		}
+	if args.awsAccountsFile != "" {
```

Comment:
Should we have a dedicated flag for CCS clusters? If you want CCS but forget to pass the file here we'll create OSD clusters.

Reply:
Good idea, make it a two-key system.